recipes/orthologer/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "3.3.0a" %}
+{% set sha256 = "787c7efa8b9f0ef598445955d95d1257159eeb599c1f3fae4327550601c1560d" %}
+
+package:
+  name: orthologer
+  version: {{ version }}
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('orthologer', max_pin="x") }}
+
+source:
+  url: https://data.orthodb.org/current/download/software/orthologer_{{ version }}.tgz
+  sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+    - cmake
+    - make
+    - llvm-openmp  # [osx]
```

Comment:
I think all deps under `make` should be moved to the `host` dependencies.

Reply:
Ok, thanks I had not completely understood the difference between build: and host: sections.

Reply:
Sorry I'm new to this environment as a contributor. I assume the review process is on a voluntary process. Is there any estimate on how long it usually takes to review a new package?